Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: clean deploy distclean
+
+# Project settings
+PROJECT = phonenumber-field
+
+# Virtual environment settings
+ENV ?= venv
+VENV = $(shell python -c "import sys; print(int(hasattr(sys, 'real_prefix')));")
+
+
+SLACK_WEBHOOK_URL ?= https://hooks.slack.com/services/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
+
+# List directories
+dist_dir = ./dist
+
+
+clean:
+	find $(clean_dirs) \( -name "*.pyc" -o -name __pycache__ -o -type d -empty \) -exec rm -rf {} + 2> /dev/null
+
+deploy: distclean
+ifneq ($(CIRCLECI),)
+	cp pypirc.conf ~/.pypirc
+endif
+	pip install build twine
+	sudo apt install python3.6-venv -y
+	python -m build
+	twine upload -r pypi-ergeon dist/*
+ifneq ($(CIRCLECI),)
+	curl -X POST --data-urlencode 'payload={"username": "circleci", "icon_url": "https://slack.global.ssl.fastly.net/d448/plugins/circleci/assets/bot_48.png", "text": "New `$(PROJECT)` version released: `$(shell python -c 'import $(PROJECT); print($(PROJECT).__version__);')`"}' $(SLACK_WEBHOOK_URL)
+endif
```

Comment:
will this command work? as I see we don't have CircleCI config for this repo

Reply:
It will work because those CIRCLECI blocks work when there is circleci. They wont be executed from local and wont break the flow.